### PR TITLE
[KRAKOW-2023] Add 2023 edition

### PR DESCRIPTION
**Pull request by MilyMilo**
Adding Kraków 2023

### CODE REVIEW

File: data/events/2023-krakow.yml [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+startdate: 2023-05-23T00:00:00+02:00
+enddate: 2023-05-23T23:59:59+02:00
+
+# Leave CFP dates blank if you don't know yet, or set all three at once.
+cfp_date_start: 2023-02-07T00:00:00+02:00
+cfp_date_end: 2023-05-07T23:59:59+02:00
+cfp_date_announce:
+
+cfp_link: "https://dodkrakow.pl/cfp2023/"
```
Comment (phrawzty) [inline data/events/2023-krakow.yml:24]: This leads to a 404. Please comment out until the CFP mechanism is ready.

image

File: data/events/2023-krakow.yml [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+registration_link: "https://eventory.cc/event/devopsdays-krakow-2023"
+
+# Location
+coordinates: "50.012427, 19.923830"
+location: "Metropolo Kraków by Golden Tulip"
+location_address: "Orzechowa 11, 30-422 Kraków"
+
+nav_elements: # List of pages you want to show up in the navigation of your page.
+  - name: registration
+    url: "https://eventory.cc/event/devopsdays-krakow-2023"
```
Comment (phrawzty) [inline data/events/2023-krakow.yml:39]: This page feels like it's not ready, but I'm not familiar with the interface. Is this expected behaviour?

image

File: data/events/2023-krakow.yml [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+
+# Leave CFP dates blank if you don't know yet, or set all three at once.
+cfp_date_start: 2023-02-07T00:00:00+02:00
+cfp_date_end: 2023-05-07T23:59:59+02:00
+cfp_date_announce:
+
+cfp_link: "https://dodkrakow.pl/cfp2023/"
+
+registration_date_start: 
+registration_date_end:
```
Comment (phrawzty) [inline data/events/2023-krakow.yml:27]: Please fill these fields in with the start and end dates of when people can register.

File: data/events/2023-krakow.yml [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+# Location
+coordinates: "50.012427, 19.923830"
+location: "Metropolo Kraków by Golden Tulip"
+location_address: "Orzechowa 11, 30-422 Kraków"
+
+nav_elements: # List of pages you want to show up in the navigation of your page.
+  - name: registration
+    url: "https://eventory.cc/event/devopsdays-krakow-2023"
+  - name: propose
+    url: "https://dodkrakow.pl/cfp2023/"
```
Comment (phrawzty) [inline data/events/2023-krakow.yml:41]: Same comment as [line 24](https://github.com/devopsdays/devopsdays-web/pull/12197/files#diff-17746e68896aef08ac7831d71531a7ecfeb5d373bc242097c9ce6093434055caR24).

**Review by phrawzty — CHANGES_REQUESTED** [review]

**Comment by phrawzty** [comment]
Hello,

This has been open for a long time without any response. Nobody responded to my query in the `#krakow` channel on Slack either.

I will close this PR for now. Feel free to re-open when you're ready.

**Comment by ivellios** [comment]
@phrawzty - I have addressed these issues in a separate PR: #12333

**Pull request closed without merging**